modules/tinymce/src/plugins/autoresize/main/ts/core/Resize.ts

```diff
@@ -29,6 +29,12 @@ const toggleScrolling = (editor: Editor, state: boolean): void => {
   }
 };
 
+interface PreviousData {
```

Comment:
Maybe name this ResizeData or something using the word Previous is a bit odd since you normally don't name types for states like that it's more a implementation detail of how you use that type that is current or previous.